wrapper.js

```diff
@@ -26,20 +26,48 @@ function setupMethods (soljson) {
   };
 
   // This calls compile() with args || cb
-  var runWithReadCallback = function (readCallback, compile, args) {
+  var runWithCallbacks = function (callbacks, compile, args) {
+    var readCallback;
+    var smtSolverCallback;
+
+    if (typeof callbacks === 'function') {
```

Comment:
Did we decide to not have this workaround in 0.6 then because we allow people to upgrade to the new syntax in 0.5?

Reply:
Yea need to rebase and will remove this